src/components/NativeSelect/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, useState, ChangeEvent } from 'react';
+import { StyledSelect } from './style';
+import chevronDown from '../../assets/icons/chevron-down-small.svg';
+import chevronUp from '../../assets/icons/chevron-up-small.svg';
+import Icon from '../Icon';
+
+export type SimpleSelectOptionType = {
```

Comment:
Can't this just be OptionType? 😛 

```suggestion
export type OptionType = {
```